sparseblas.tex

```diff
@@ -1085,22 +1085,21 @@ \subsection{Single-Stage API for Operations with known Output}
 // auto B_obj = matrix_handle(B, allocator);
 // auto D_obj = matrix_handle(D, allocator);
 
-sparse_multiply_state_t state(allocator);
-sparse_multiply_inspect(policy, state, 
+operation_state_t state(allocator);
+multiply_inspect(policy, state, 
```

Comment:
if we use only one type `operation_state_t` for all kinds of operation, we will lose the possibility to allow user to use them separately easily.
If we still keep them individually in the first place, have a structure to collect all state should relatively be easier than the other way.